script.js

```diff
@@ -0,0 +1,61 @@
+function toggleHiddenElements() {
+  const elements = [
+    '.menu-bars',
+    'main',
+    'footer',
+    'h1',
+  ];
+
+  elements.forEach((element) => {
+    document.querySelector(element).classList.toggle('hide');
+  });
+}
+
+function toggleShownElements() {
+  const elements = [
+    '#desktop-nav',
+    '#desktop-nav ul',
+  ];
+
+  elements.forEach((element) => {
+    document.querySelector(element).classList.toggle('show');
+  });
+}
+
+function removeCloseBtn() {
+  const closeBtn = document.querySelector('.close-btn');
+  closeBtn.parentNode.removeChild(closeBtn);
+}
+
+function closeMenu() {
+  document.querySelector('header').classList.remove('menu-visible');
+  toggleShownElements();
+  toggleHiddenElements();
+  removeCloseBtn();
+}
```

Comment:
- [ ] Great job on implementing all the changes, but I noticed that the menu close button is not working well, so kindly Fix it because it is important to do the functionality for the menu to meet all the requirements. 💪🏻